client/src/pages/Login/Login.tsx

```diff
@@ -37,26 +38,25 @@ export default function Login(): JSX.Element {
 
   return (
     <Grid container component="main" className={classes.root}>
-      <Grid item xs={12} sm={8} md={7} elevation={6} component={Paper} square>
-        <Box
-          display="flex"
-          justifyContent="space-between"
-          alignItems="flex-start"
-          flexDirection="column"
-          className={classes.authWrapper}
-        >
+      <Grid item xs={12} sm={12} elevation={6} component={Paper}>
+        <Box className={classes.authWrapper}>
           <AuthHeader linkTo="/signup" asideText="Don't have an account?" btnText="Create account" />
-          <Box width="100%" maxWidth={450} p={3} alignSelf="center">
+          <Box minWidth={300} p={3} alignSelf="center" className={classes.formContainer}>
             <Grid container>
               <Grid item xs>
-                <Typography className={classes.welcome} component="h1" variant="h5">
+                <Typography className={classes.welcome} style={{ fontWeight: 700 }} variant="h4">
```

Comment:
Please avoid the usage of inline styles, use a CSS class instead

Ditto where applicable

Reply:
Done